source/includes/examples/ex-eval-output.rst

```diff
@@ -1,9 +1,12 @@
+To get output suitable for automated parsing, use
+``EJSON.stringify()``.
```

Comment:
Just a heads up, no need to change anything – while this is perfectly true and correct advice, we are planning to provide a less cumbersome way of getting parse-able results as part of [MONGOSH-1249](https://jira.mongodb.org/browse/MONGOSH-1249)).

Reply:
Cool, thanks for the info